docs/components/card-catalog.md

```diff
@@ -420,8 +420,11 @@ Set the `isLoading` prop to `true` to enable the loading state.
 
 ## Slots
 
-- `body` - The body of the card catalog, we expect this to be an array of `KCatalogItem` components.
-This should be used instead of the `items` property.
+Both the `title` & `description` of the card items as well as the entire catalog `body` are slottable.
```

Comment:
```suggestion
Both the `title` & `description` of the card items as well as the entire `KCardCatalog` `body` are slottable.
```

suggestion: verbiage update

Reply:
I am going to leave this the same since we plan on changing the component name anyway.